internal/pkg/scaffold/project/go_mod.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package project
+
+import (
+	"fmt"
+
+	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/input"
+	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/internal/deps"
+)
+
+const GoModFile = "go.mod"
+
+type GoMod struct {
+	input.Input
+}
+
+func (s *GoMod) GetInput() (input.Input, error) {
+	if s.Path == "" {
+		s.Path = GoModFile
+	}
+	s.TemplateBody = goModTmpl
+	return s.Input, nil
+}
+
+const goModTmpl = `module {{ .Repo }}
+
+go 1.11
+
+require (
+	contrib.go.opencensus.io/exporter/ocagent v0.4.9 // indirect
+	github.com/Azure/go-autorest v11.5.2+incompatible // indirect
+	github.com/appscode/jsonpatch v0.0.0-20190108182946-7c0e3b262f30 // indirect
+	github.com/coreos/prometheus-operator v0.26.0 // indirect
```

Comment:
There is a PR to update this -> https://github.com/operator-framework/operator-sdk/pull/1280

Reply:
Added a `replace` entry for `prometheus-operator`.